rwengine/src/render/GameRenderer.hpp

```diff
@@ -140,8 +140,9 @@ class GameRenderer {
     void setupRender();
     void renderPostProcess();
 
-    Renderer* getRenderer() {
-        return renderer.get();
+    Renderer*& getRenderer() {
```

Comment:
This should either return a reference or a pointer. If you don't want to update all calls to this function I would be happy with a comment on the function in the header that the pointer is owned by the GameRenderer.